analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -242,20 +244,67 @@ class Pub(
             log.info { "Successfully read lockfile." }
 
             val parsePackagesResult = parseInstalledPackages(lockFile, labels)
+            val dependenciesScope = parseScope("dependencies", manifest, lockFile, parsePackagesResult.packages, labels)
+
+            if (containsFlutterSdk(workingDir)) {
+                // Run Gradle dependency resolution. This is required because for Flutter projects, Pub needs to run
+                // first to generate the local.properties file by using `flutter pub get`.
+                val gradleFactory = Gradle.Factory()
+                val gradleDefinitionFiles =
+                    findManagedFiles(analysisRoot, listOf(gradleFactory)).getOrDefault(gradleFactory, emptyList())
+
+                if (gradleDefinitionFiles.isNotEmpty()) {
+                    println("Found ${gradleDefinitionFiles.size} ${gradleFactory.managerName} project(s) at:")
```

Comment:
Forgot to turn the `println()` statements into `log()` ones? (also in line 263)

Reply:
No, this is intended.
See the notes at the end of this comment: https://github.com/oss-review-toolkit/ort/pull/4831#issue-1075626935.

Reply:
However, @fviernau is right that we should not call `println()` in library code, as we cannot be sure to have a console attached. So we might need to turn this into a `log` after all.

Reply:
Thanks for the explanation. Replaced it with info logs.